src/Data/Nat/Properties.agda

```diff
@@ -1473,6 +1498,16 @@ m∸[m∸n]≡n {suc m} {suc n} (s≤s n≤m) = begin-equality
 *-distrib-∸ : _*_ DistributesOver _∸_
 *-distrib-∸ = *-distribˡ-∸ , *-distribʳ-∸
 
+2m≢1+2n :  ∀ m n → 2 * m ≢ suc (2 * n)
+2m≢1+2n m n 2m≡1+2n =  m>1⇒m*n≢1 {2} {m ∸ n} n<1+n 2[m-n]≡1
```

Comment:
I don't think it's necessary to use a `where` block here.

Reply:
Removed "where".